src/test/java/io/ebean/dbmigration/DbMigrationTest.java

```diff
@@ -1,31 +1,142 @@
 package io.ebean.dbmigration;
 
 import io.ebean.BaseTestCase;
+import io.ebean.SqlQuery;
+import io.ebean.SqlRow;
+import io.ebean.SqlUpdate;
+import io.ebean.Transaction;
+import io.ebean.dbmigration.ddl.DdlRunner;
+
+import org.junit.Ignore;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.List;
+
+import javax.persistence.PersistenceException;
+
 public class DbMigrationTest extends BaseTestCase {
 
   private static final Logger logger = LoggerFactory.getLogger(DbMigrationTest.class);
 
-  //@Ignore
-  @Test
-  public void writeCurrent() {
 
-    logger.info("start");
+  private int runScript(boolean expectErrors, String scriptName) throws IOException {
+    try (InputStream stream = getClass().getResourceAsStream("/dbmigration/migrationtest/" + server().getPluginApi().getDatabasePlatform().getName()+"/" + scriptName);
+      java.util.Scanner s = new java.util.Scanner(stream)) {
+      s.useDelimiter("\\A");
+      if (s.hasNext()) {
+        return runScript(expectErrors, s.next(), scriptName);
+      }
+    }
+    return 0;
+  }
+
+  private int runScript(boolean expectErrors, String content, String scriptName) {
+
+    DdlRunner runner = new DdlRunner(expectErrors, scriptName);
+
+    Transaction transaction = server().createTransaction();
+    Connection connection = transaction.getConnection();
+    try {
+      if (expectErrors) {
+        connection.setAutoCommit(true);
+      }
+      int count = runner.runAll(content, connection);
+      if (expectErrors) {
+        connection.setAutoCommit(false);
+      }
+      transaction.commit();
+      return count;
+
+    } catch (SQLException e) {
+      throw new PersistenceException("Failed to run script", e);
+
+    } finally {
+      transaction.end();
+    }
+  }
+  @Test
+  public void testRunMigration() throws IOException {
+    // first clean up previously created objects
+    runScript(true, "drop table migtest_e_ref;\n","test");
+    runScript(true, "drop table migtest_e_basic;\n"
+        + "drop table migtest_e_history;\n"
+        + "drop table migtest_e_ref;\n"
+        + "drop table migtest_e_ref cascade;\n"
+        + "drop table migtest_e_user;\n"
+        + "drop table migtest_e_history;\n"
+        + "drop table migtest_e_history cascade;\n" // pg
+        + "drop table migtest_e_history_history cascade;\n" // pg
+        + "drop sequence migtest_e_basic_seq;\n"
+        + "drop sequence migtest_e_history_seq;\n"
+        + "drop sequence migtest_e_ref_seq;\n"
+        + "drop sequence migtest_e_user;\n"
+        + "drop sequence migtest_e_history;\n"
+        , "cleanup");
+    
+    
+    runScript(false, "1.0__initial.sql");
```

Comment:
This test tries to apply the migrations and makes some simple tests if migrations did work